exchange/utils.go

```diff
@@ -297,13 +297,27 @@ func getBidderParamsForBidder(bidderParamsInReqExt map[string]map[string]json.Ra
 	return params, nil
 }
 
-func getExtJson(req *openrtb2.BidRequest, unpackedExt *openrtb_ext.ExtRequest) (json.RawMessage, error) {
+func getBidderRequestExtJson(req *openrtb2.BidRequest, unpackedExt *openrtb_ext.ExtRequest, bidder string) (json.RawMessage, error) {
 	if len(req.Ext) == 0 || unpackedExt == nil {
 		return json.RawMessage(``), nil
 	}
 
 	extCopy := *unpackedExt
 	extCopy.Prebid.SChains = nil
+	extCopy.Prebid.AlternateBidderCodes = nil
+
+	//fill request.ext.prebid.alternatebiddercode to have data only of the current bidder
+	if unpackedExt.Prebid.AlternateBidderCodes != nil && len(unpackedExt.Prebid.AlternateBidderCodes.Bidders) != 0 {
```

Comment:
Do we want to pass `openrtb_ext.ExtAlternateBidderCodes{Enabled: true/false}` to the bidders that are not present in the list req.ext?.

Current changes do not pass it as I don't find any use case for the same.